[ROS2] windows build error for the ros2 branch

[accidentally posted this](https://github.com/ros-perception/image_common/issues/189) over on `image_common`...but the problem is actually with the `image_geometry` package in this repo.

```
-- Selecting Windows SDK version 10.0.18362.0 to target Windows 10.0.18363.
-- Found ament_cmake_python: 0.9.8 (C:/opt/ros/foxy/x64/share/ament_cmake_python/cmake)
-- Found ament_cmake_ros: 0.9.0 (C:/opt/ros/foxy/x64/share/ament_cmake_ros/cmake)
-- Using PYTHON_EXECUTABLE: C:/opt/ros/foxy/x64/python.exe
-- OpenCV ARCH: x64
-- OpenCV RUNTIME: vc16
-- OpenCV STATIC: OFF
-- Found OpenCV 3.4.6 in C:/opencv/x64/vc16/lib
-- You might need to add C:\opencv\x64\vc16\bin to your PATH to be able to run your applications.
-- Found sensor_msgs: 2.0.3 (C:/opt/ros/foxy/x64/share/sensor_msgs/cmake)
-- Using all available rosidl_typesupport_c: rosidl_typesupport_fastrtps_c;rosidl_typesupport_introspection_c
-- openCV libraries cmake variable: opencv_calib3d;opencv_core;opencv_highgui;opencv_imgproc
-- Found rosidl_adapter: 1.2.0 (C:/opt/ros/foxy/x64/share/rosidl_adapter/cmake)
-- Using all available rosidl_typesupport_cpp: rosidl_typesupport_fastrtps_cpp;rosidl_typesupport_introspection_cpp
-- Found gtest sources under 'C:/opt/ros/foxy/x64/src/gtest_vendor': C++ tests using 'Google Test' will be built
-- Configuring done
-- Generating done
-- Build files have been written to: D:/code/ros/rolling/build/image_geometry
Microsoft (R) Build Engine version 16.9.0+5e4b48a27 for .NET Framework
Copyright (C) Microsoft Corporation. All rights reserved.

     Creating library D:/code/ros/rolling/build/image_geometry/Release/image_geometry.lib and object D:/code/ros/rolling/build/image_geometry/Release/image_geometry.exp
pinhole_camera_model.obj : error LNK2019: unresolved external symbol "void __cdecl cv::error(int,class std::basic_string<char,struct std::char_traits<char>,class std::allocator<char> > const &,char const *,char const *,int)" (?error@cv@@YAXHAEBV?$basic_string@DU?$char_traits@D@std@@V?$allocator@D@2@@std@@PEBD1H@Z) referenced in function "public: __cdecl cv::Mat::Mat(int,int,int,void *,unsigned __int64)" (??0Mat@cv@@QEAA@HHHPEAX_K@Z) [D:\code\ros\rolling\build\image_geometry\image_geometry.vcxproj]
D:\code\ros\rolling\build\image_geometry\Release\image_geometry.dll : fatal error LNK1120: 1 unresolved externals [D:\code\ros\rolling\build\image_geometry\image_geometry.vcxproj]
  gtest.vcxproj -> D:\code\ros\rolling\build\image_geometry\gtest\Release\gtest.lib
  gtest_main.vcxproj -> D:\code\ros\rolling\build\image_geometry\gtest\Release\gtest_main.lib
```

I added a print to see the CMake_LIBRARIES variable to make sure it was actually linking to it, you can see the output above - copied here for easy reading: 
```
-- openCV libraries cmake variable: opencv_calib3d;opencv_core;opencv_highgui;opencv_imgproc
```

--- a/image_geometry/CMakeLists.txt
+++ b/image_geometry/CMakeLists.txt
@@ -18,7 +18,9 @@ endif()
 find_package(OpenCV REQUIRED COMPONENTS calib3d core highgui imgproc)
 find_package(sensor_msgs REQUIRED)
 
-include_directories(include)
+include_directories(${CMAKE_CURRENT_SOURCE_DIR}/include
+  ${OpenCV_INCLUDE_DIRS}
+)
 
 # add a library
 add_library(${PROJECT_NAME}

